app/controllers/users/passwords_controller.rb

```diff
@@ -8,11 +8,15 @@ class Users::PasswordsController < Devise::PasswordsController
 
   def create
     user = User.find_for_authentication(resource_params.permit(:login))
+    if user.nil? || user.new_record?
+      flash[:notice] = t(".user_not_found")
```

Comment:
We should make this notice an error instead,  since the field error is no longer in effect.